#!/usr/bin/env python
PACKAGE = "quanergy_client_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_gps", str_t, 0, "Coord X", "0")
gen.add("y_gps", str_t, 0, "Coord Y", "0")
gen.add("z_gps", str_t, 0, "Coord Z", "0")


exit(gen.generate(PACKAGE, "quanergy_client_ros", "coordenadas_ref"))
